frontend/src/components/common/application-error-alert/style.module.scss

```diff
@@ -0,0 +1,27 @@
+/*!
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+.exclamation {
+  font-size: 2.3em;
+  line-height: 53px;
```

Comment:
Does it make sense to mix absolute values (in pixels) and relational values (in em)?